jsonParser_test.go

```diff
@@ -0,0 +1,9 @@
+package crossrefindexer
+
+import (
+	"testing"
+)
+
+func Test_JsonParser(t *testing.T) {
+	JsonParser()
```

Comment:
You are really not testing anything here?

Reply:
That's correct, I am only using testing as alternative to main func.